src/components/Label/Label.tsx

```diff
@@ -1,19 +1,57 @@
 import React from 'react'
 import FormLabel from 'react-bootstrap/FormLabel'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
 interface Props {
   /** Text to display in label */
   text: string
-  /** Ties labels to input */
+  /** Title of the label. */
+  title?: string | '' // Use on required input labels to override default required title
+  /** Give option to disable  */
   htmlFor?: string | ''
+  /** Disables title */
+  disableTitle?: boolean | false
```

Comment:
What situations are you thinking we would use the `disableTitle` prop in?

Reply:
Thank you for all the constructive criticism! I was thinking of giving more control to the user incase there are many labels and the developer sees the titles aren't necessary but I also thought it might be redundent. 

I am going to think through these contributions better and avoid anything I think might be out of my league. Thanks again for these resolutions and I will try to focus more on bugs and some necessary simple features for now! :grin: